project_clusters.go

```diff
@@ -162,10 +175,11 @@ func (s *ProjectClustersService) AddCluster(pid interface{}, opt *AddClusterOpti
 // GitLab API docs:
 // https://docs.gitlab.com/ee/api/project_clusters.html#edit-project-cluster
 type EditClusterOptions struct {
-	Name               *string                        `url:"name,omitempty" json:"name,omitempty"`
-	Domain             *string                        `url:"domain,omitempty" json:"domain,omitempty"`
-	EnvironmentScope   *string                        `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
-	PlatformKubernetes *EditPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	Name                *string                        `url:"name,omitempty" json:"name,omitempty"`
+	Domain              *string                        `url:"domain,omitempty" json:"domain,omitempty"`
+	EnvironmentScope    *string                        `url:"environment_scope,omitempty" json:"environment_scope,omitempty"`
+	PlatformKubernetes  *EditPlatformKubernetesOptions `url:"platform_kubernetes_attributes,omitempty" json:"platform_kubernetes_attributes,omitempty"`
+	ManagementProjectID *string                        `url:"management_project_id,omitempty" json:"management_project_id,omitempty"`
```

Comment:
Same... Should be above `PlatformKubernetes`.

Reply:
Done